inference-engine/src/transformations/src/transformations/common_optimizations/common_optimizations.cpp

```diff
@@ -163,6 +164,9 @@ bool ngraph::pass::CommonOptimizations::run_on_function(std::shared_ptr<ngraph::
     manager.register_pass<ngraph::pass::ConstantFolding>();
     // need to convert to Gather-1 until plugins do not support Gather-7
     manager.register_pass<ngraph::pass::ConvertGather7ToGather1>();
+    // to enhance operation maintenance some plugins can choose to
+    // work with a single general Gather op with batch_dims
```

Comment:
nit: we don't have comments in the similar transformations above. So, let's remove the comment here as well.

@GlebKazantaev , what do you think?

Reply:
removed this comment